.travis.yml

```diff
@@ -57,7 +57,8 @@ script:
   - DCO_VERBOSITY=-q script/validate/dco
   - script/validate/fileheader
   - GOOS=linux script/setup/install-dev-tools
-  - script/validate/vendor
+# #2135: dirty cri vendor because of circular dependency. will be fixed immediately in a follow-up PR.
```

Comment:
cc @Random-Liu 
Is this correct way?

Reply:
I believe the correct way is to update it in a fork and point the vendor to your fork. We should never do a merge into master without validated vendors.

Reply:
updated